### ISSUE TITLE
Special Characters cannot be used in package path

### ISSUE THREAD

**Issue by makefu** [victorrjimenezz/PS4-4PT#11]
Right now download fails of packages with a special character in the path (most notably space). To allow integration, the path should be url-encoded before requesting.

example repo:

```
name: Game extras                                                                                                        
'Addon [01.00]':
  pkgPath: "pkg/Addon [v1.00] [DLC].pkg"
  type: MISC
```

**Comment by victorrjimenezz** [comment]
Hi!! Thank you for reporting this bug! Completely agree, will fix this right now and bundle it with the upcoming update (release later today or early tomorrow) :)

**Comment by makefu** [comment]
awesome, thanks for quick response!

**Comment by victorrjimenezz** [comment]
just fixed it and got it working :) Thank you for sharing this, the commit has been pushed and the fix will be released in version 2.35
